charts/opentelemetry-demo/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
 name: opentelemetry-demo
-version: 0.20.0
+version: 0.20.1
```

Since this is a breaking change, bump the minor version instead of patch.

```suggestion
version: 0.21.0
```